yb-voyager/cmd/miginfo.go

```diff
@@ -16,60 +16,45 @@ limitations under the License.
 package cmd
 
 import (
-	"encoding/json"
 	"fmt"
-	"os"
-	"path/filepath"
 
 	log "github.com/sirupsen/logrus"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/metadb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 )
 
-type MigInfo struct {
```

Comment:
The original intent with which we introduced `MigInfo` is now fulfilled by the `MigrationStatusRecord`. So let's get rid of the `MigInfo` by simply dumping the `srcdb.Source` in the `MSR`.